<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>doctor</class>
 <widget class="QDialog" name="doctor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>330</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add New Doctor</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>330</width>
     <height>300</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: transparent;
border-radius: 0px;</string>
    </property>
    <property name="text">
     <string>Doctor Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_doctor_name">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>260</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(55, 64, 71);
border-radius: 6px;
border: 1px solid #18B1A8;</string>
    </property>
    <property name="placeholderText">
     <string>Enter Doctor Name</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: transparent;
border-radius: 0px;</string>
    </property>
    <property name="text">
     <string>Specialty </string>
    </property>
   </widget>
   <widget class="QComboBox" name="cmb_specialty">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>150</y>
      <width>261</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox
{
 background-color: rgb(255, 255, 255);
color: rgb(55, 64, 71);
border-radius: 6px;
border: 1px solid #18B1A8;
}



</string>
    </property>
    <property name="placeholderText">
     <string>Select</string>
    </property>
    <item>
     <property name="text">
      <string>IOL</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Retinal</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Cornea</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="btn_savedoctor">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>210</y>
      <width>260</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(24, 177, 168);
border-radius: 6px;
border: 1px solid #18B1A8;
font: 500 11pt &quot;Ubuntu&quot;;
color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string>Submit</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
